src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Media/GradientBrushTests/RadialGradientBrushPage.xaml.cs

```diff
@@ -0,0 +1,263 @@
+﻿#if !NETFX_CORE
+using System;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Windows.Foundation;
+using Windows.Foundation.Metadata;
+using Windows.UI;
+using Windows.UI.Composition;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Media.Imaging;
+using Uno.UI.Samples.Controls;
+
+namespace MUXControlsTestApp
+{
+	[Sample("GradientBrush", Name = "RadialGradientBrush")]
+	public sealed partial class RadialGradientBrushPage : Page
+	{
+		public RadialGradientBrush DynamicGradientBrush;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5417387)